klaas/scripts/split_data.py

```diff
@@ -0,0 +1,55 @@
+import click
+import numpy as np
+
+from ..io import read_data
+import warnings
+
+
+@click.command()
+@click.argument('input_path', type=click.Path(exists=True, file_okay=True))
+@click.argument('output_basename')
+@click.option(
+    '--fraction', '-f', multiple=True, type=float,
+    help='Fraction of events to use for this part'
+)
+@click.option('--name', '-n', multiple=True, help='name for one dataset')
+@click.option('--key', '-k', help='Name for the hdf5 group in the output', default='data')
+@click.option(
+    '--fmt', type=click.Choice(['csv', 'hdf5']), default='hdf5',
+    help='The output format',
+)
+def main(input_path, output_basename, fraction, name, key, fmt):
+    '''
+    Split dataset in INPUT_PATH into multiple parts for given fractions and names
```

Comment:
what will be the names of the output files if you suply more than two fractions?